Spotkanie3_Zadania/String/String/Program.cs

```diff
@@ -0,0 +1,267 @@
+            Console.WriteLine($"Comparing whether \"{word1}\" is equal to \"{word2}\":\t {result1}");
+        }
+
+        //returns the position of the specified character in the string
+        static void ReturnPositionOfIndicatedCharacter(string word)
+        {
+            char letterLookingFor;
+            while (true)
+            {
+                Console.Write("We are looking for the index of the first occurrence of the letter you provided, dear user: ");
+                if (char.TryParse(Console.ReadLine(), out letterLookingFor))
+                {
+                    if (Char.IsNumber(letterLookingFor))
+                    {
+                        break;
+                    }
+                    else Console.WriteLine("Dear User, you have not entered a letter but a number, please try again.");
+                }
+                else Console.WriteLine("Dear User, you have not entered a letter, please try again.");
+            }
+            Console.Write($"We are looking for the index of the first occurrence of the letter '{letterLookingFor}': ");
+            if (word.IndexOf(letterLookingFor) < 0) Console.WriteLine($"The specified letter \"{letterLookingFor}\" is missing in the string: {word}");
+            else Console.WriteLine(word.IndexOf(letterLookingFor));
+        }
+
+        //returns substring of a string
+        static void ReturnsSubstringOfString(string word)
+        {
+            Console.Write("We print now substring which start from index what you want position: ");
+            int startIndex = EnterNumber();
+
+            Console.Write("We print now substring which end in what you want position: ");
+            int endIndex = EnterNumber();
+
+            Console.WriteLine($"We print now substring which start from index {startIndex} and end in {endIndex} position: {word.Substring(startIndex, endIndex)} ");
+        }
+
+        static int EnterNumber()
+        {
+            int numberCandidate;
+            while (true)
+            {
+                if (int.TryParse(Console.ReadLine(), out numberCandidate)) break;
+                else Console.Write($"Dear User, this is not a number, please try again: ");
+            }
+            return numberCandidate;
+        }
+
+
+        //splits the string into substring
+        static void SplitsString(string word)
+        {
+            Console.WriteLine($"We print now our string but splits a letter: ");
+            foreach (string letter in word.Split("")) Console.Write(letter);
+        }
+
+        public static void EndProgramme()
+        {
+            while (true)
+            {
+                Console.WriteLine($"Dear user do you want to continue: ");
+                Console.WriteLine("If 'Yes' enter: '1'.");
+                Console.WriteLine("If 'No' enter: '2'.");
+                GoOutOrStay(EnterNumber());
+                break;
+            }
+        }
+
+        public static void GoOutOrStay(double ifExit)
```

Comment:
Jesli w nazwie funkcji/metody, musisz uzyc slowa `or, and, also` itp, to juz jest sygnal, ze funkcja robi co najmniej dwie rozne rzeczy.
Podzielilbym ja na osobne, jedna odpowiedziala na przygotowanie appki do stanu poczatkowego, a druga za zakonczenie pracy programu.

Reply:
Dobra tutaj się zatrzymam i dokończę jutro czy coś 🍡

Reply:
Rozdzieliłem i troszeczkę pozmieniałem, ale mam nadzieję że jest ok.